internal/cmd/beta/alb/credentials/credentials.go

```diff
@@ -0,0 +1,33 @@
+package credentials
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/alb/credentials/create"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/alb/credentials/delete"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/alb/credentials/describe"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/alb/credentials/list"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/alb/credentials/update"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+)
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "credentials",
```

Comment:
The name observability-credentials fits a bit better here. Because the credentials are used to connect observability and the application load-balancer. In general I think the commands should be more similar to load-balancer observability-credentials to keep it consistent.
```suggestion
		Use:   "observability-credentials",
```